src/components/form/row.vue

```diff
@@ -13,7 +13,13 @@ except according to the terms contained in the LICENSE file.
   <tr class="form-row">
     <td class="name">
       <link-if-can :to="primaryFormPath(form)">
-        <span>{{ form.nameOrId() }}</span><span class="icon-angle-right"></span>
+        <span v-if="form.state == 'closed'" class="icon-times-circle text-danger form-icon"
```

Comment:
We tend not to use Boostrap helper classes like `text-danger`. Instead, we specify those styles in the CSS, often selecting on some other class. (Here, that other class would probably be `icon-times-circle`.) We often use Sass variables, and if the styles are complex, we use a Sass mixin. That can lead to more complex CSS, but it's nice that it simplifies the template and arguably makes things more semantic.